### PR TITLE
Fix Makefile

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| License       | MIT

Alternative to #202, reverts #200 and #201.

### CODE REVIEW

File: symfony/framework-bundle/3.3/Makefile [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 .PHONY: cache-warmup
 
 serve_as_sf: sf_console
-ifeq ($(CONSOLE), '')
+ifndef CONSOLE
 	@${MAKE} serve_as_php
 endif
-	@$(CONSOLE) | grep server:start > /dev/null || ${MAKE} serve_as_php
-	@$(CONSOLE) server:start
+	@bin/console list | grep server:start > /dev/null || ${MAKE} serve_as_php
```
Comment (nicolas-grekas) [inline symfony/framework-bundle/3.3/Makefile]: let's not assume `list` is the default command

File: symfony/framework-bundle/3.3/Makefile [hunk 2]
```diff
@@ -1,31 +1,31 @@
-CONSOLE := $(shell printf %q "`which bin/console`")
+CONSOLE := $(shell which bin/console)
```
Comment (fabpot) [inline symfony/framework-bundle/3.3/Makefile]: That's what we had before, but that does not work well if there is a space in your directory names.

Comment (fabpot) [reply symfony/framework-bundle/3.3/Makefile]: ah, sorry, I see that you've changed the code using $CONSOLE as well. nvm

Comment (ElectricMaxxx) [reply symfony/framework-bundle/3.3/Makefile]: Why not `test -e` when using it bool'sch only?

**Pull request merged**